src/loadsave.cpp

```diff
@@ -537,6 +538,63 @@ bool findLastSave()
 	return foundMP | foundCAM;
 }
 
+static WzString suggestSaveName(const char *NewSaveGamePath)
+{
+    const WzString levelName = getLevelName();
+    const std::string cheatedSuffix = Cheated ? _("cheated") : "";
+    char saveNamePartial[64] = "\0";
+
+    if (bLoadSaveMode == SAVE_MISSIONEND || bLoadSaveMode == SAVE_INGAME_MISSION)
+    {
+        std::string campaignName;
+        if (levelName.startsWith("CAM_1"))
+        {
+            campaignName = "Alpha";
+        }
+        else if (levelName.startsWith("CAM_2"))
+        {
+            campaignName = "Beta";
+        }
+        else if (levelName.startsWith("CAM_3"))
```

Comment:
Also take into consideration here that offworld mission names start with "SUB_".